### PR TITLE
core: return grant in create-grant API response

### PR DESCRIPTION
**Pull request by jeffomatic**
This makes the `/create-authorization-grant` RPC consistent in behavior to all other `/create-`-prefixed endpoints.

### CODE REVIEW

**Comment by jeffomatic** [comment]
slight revision: removed temp var in grant.go

File: core/grants.go [hunk 1]
```diff
@@ -47,7 +47,7 @@ func (a *API) createGrant(ctx context.Context, x apiGrant) error {
 	// so we should do our equality comparisons accordingly.
 	guardData, err := json.Marshal(x.GuardData)
 	if err != nil {
-		return errors.Wrap(err)
+		return nil, errors.Wrap(err)
 	}
 
 	g := authz.Grant{
```
Comment (tessr) [inline core/grants.go]: Why not just return `g` if `err == nil` at the end of this, instead of modifying `StoreGrant`?

Comment (jeffomatic) [inline core/grants.go]: I want to return the grant including the `CreatedAt` timestamp, which StoreGrant generates or retrieves.

Comment (tessr) [inline core/grants.go]: Ah, got it 👍

**Comment by tessr** [comment]
LGTM

**Pull request merged**